deps/g3dlite/include/G3D/Vector3.h

```diff
@@ -116,6 +116,11 @@ class Vector3 {
 
     // assignment and comparison
     Vector3& __fastcall operator= (const Vector3& rkVector);
+    /* requried as of C++ 11 */
+    #if __cplusplus >= 201103L
```

Comment:
its not needed c++17 enabled

Reply:
problem persists with -DCMAKE_CXX_FLAGS="-Werror -std=c++17"

Reply:
https://en.cppreference.com/w/cpp/language/default_constructor

![image](https://user-images.githubusercontent.com/147092/81572071-091bf180-93a3-11ea-8581-ff563c9a5ce7.png)

@Viste